fem/sidredatacollection.cpp

```diff
@@ -489,6 +504,73 @@ createMeshBlueprintTopologies(bool hasBP, const std::string& mesh_name)
 }
 
 // private method
+#ifdef MFEM_USE_MPI
+void SidreDataCollection::createMeshBlueprintAdjacencies(bool hasBP)
+{
+   ParMesh *pmesh = dynamic_cast<ParMesh*>(mesh);
+
+   const int GRP_SZ = 35;
+   char group_str[GRP_SZ];
+
+   // TODO(JRC): Separate this out into group hierarchy setup and data allocation
+   // stages like all of the other "createMeshBlueprint*" functions.
```

Comment:
I apologize for using confusing wording and for making a few incorrect remarks in my initial comment.  The following is the situation as I understand it.  

This code will not work when `hasBP == true` because it will attempt to recreate existing data groups/views.  This pull request should only be merged if this deficiency is permissible for this initial implementation.  If not, I can work to update the implementation so that it properly only creates data groups/views when `hasBP == false` (similar to how the other `createMeshBlueprint` functions work).